### ISSUE TITLE
[1337x] Exception (1337x): Could not load file or assembly 'Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The module was expected to contain an assembly manifest.: Could not load file or assembly 'Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The module was expected to contain an assembly manifest. (Test)

### ISSUE THREAD

**Issue by langdonholmes** [Jackett/Jackett#6170]
I get this error with all trackers except for RARBG and YTS. I am running on a raspberry pi with Raspbian.

```
2019-10-13 13:56:23.0100 Error Jackett.Common.IndexerException: Exception (1337x): Could not load file or assembly 'Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The module was expected to contain an assembly manifest. ---> System.BadImageFormatException: Could not load file or assembly 'Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The module was expected to contain an assembly manifest.
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query)
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Server.Controllers.ResultsController.Torznab(TorznabRequest request) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 367 Jackett.Common.IndexerException: Exception (1337x): Could not load file or assembly 'Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The module was expected to contain an assembly manifest. ---> System.BadImageFormatException: Could not load file or assembly 'Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The module was expected to contain an assembly manifest.
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query)
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Server.Controllers.ResultsController.Torznab(TorznabRequest request) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 367
```
 Jackett Version 0.11.760.0

**Comment by garfield69** [comment]
sounds like you have run afoul of the updater issue #5208
your best bet is to download and install the latest package from https://github.com/Jackett/Jackett/releases

if the updated continues to be a problem then enabling the disable-auto-update option from your jackett dashboard would be a way to prevent further issues until it is resolved.
don't forget to click on the apply-server-settings button to save your settings.

**Comment by garfield69** [comment]
as a aside you may be wondering why yts and rarbg work and most other indexers do not?
that is because yts and rarbg are indexers written in c#, and the ones that are failing are using the cardigann processor (indexer written in yaml) which needs that missing Microsoft.CSharp module.

**Comment by langdonholmes** [comment]
Thanks so much @garfield69 . I really appreciate your help. My family has very much been enjoying this setup.

**Comment by garfield69** [comment]
fix is available with stable jackett 0.12.898. but you will need to install manually and run the systemd install script anew.

1. open a terminal and shutdown the jackett service using `systemctl stop jackett.service`
2. download and extract the linux package that applies to your platform from https://github.com/Jackett/Jackett/releases/tag/v0.12.898
3. copy or move/replace all the files from the Jackett folder in the downloaded/extracted package to where ever you have previously put the jackett binaries. Typical locations are `/opt/Jackett` or `~/Jackett`
4. cd to your Jackett binary folder and install the service with `sudo ./install_service_systemd.sh` 
this will start the service (if it ran successfully)
5. access your jackett dashboard with your browser and untick the Disable auto Update check box and press the Apply Server Settings button.
